lib/commands/job.js

```diff
@@ -0,0 +1,7 @@
+exports.command = 'job <command>'
+exports.desc = 'Manage jenkins jobs'
+exports.builder = function (yargs) {
```

Comment:
Again, here is a clearer example that the code cleanness would benefit from using arrow functions.